polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -292,6 +292,9 @@ pub mod pallet {
 
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
+
+		/// Configure the fees.
+		type FeeManager: FeeManager;
```

Comment:
I am in favor of re-exposing `FeeManager` from `XcmExecutor` type instead of creating yet another configuration type here.

I don't see why `pallet-xcm` would be configured with fee manager different than the `xcm-executor`. The benefit outweighs the overhead cost of configuring it twice and thus also the risk of misconfiguring it.

`type XcmExecutor` here can take a `trait  FeeManager` bound.

`xcm-executor` implementation can `impl FeeManager` where it would simply re-expose its inner configured `type FeeManager`.

This will help with ergonomics and allow most runtimes to ingest this change with no breakages and needs to adjust configs.

Reply:
Done!